src/action.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { Action } from "./action";
+import { Step, SwapAction } from "./cnd/swaps_payload";
+import { Swap } from "./swap";
+
+const swapAction: SwapAction = { name: Step.Redeem, href: "/redeem" };
+
+describe("Action", () => {
```

Comment:
I appreciate the effort for writing tests but I am not sure how much risk they are eliminating, esp. given their verbosity and mocking efforts.

With the suggestion below of letting `cnd` handle the "action is no longer recommended" scenario, I don't think we need these tests.